docker/Dockerfile.k8s

```diff
@@ -5,7 +5,7 @@ FROM python:2.7-alpine3.9 as scalyr-dependencies
 MAINTAINER Scalyr Inc <support@scalyr.com>
 
 RUN apk --update add build-base python-dev gcc
-RUN pip --no-cache-dir install --root /tmp/dependencies ujson yappi
+RUN pip --no-cache-dir install --root /tmp/dependencies ujson yappi tlslite-ng
```

Comment:
> FROM **python:2.7**-alpine3.9
> ...
> RUN pip --no-cache-dir install --root /tmp/dependencies ujson yappi **tlslite-ng**

Is this required given that we know we'll be running on 2.7?

Reply:
Oh, this should have been deleted (since we are including src code in 3rd party libs).